tests/test-files/transform/Elm-0.18/Examples.formatted.elm

```diff
@@ -2,31 +2,15 @@ module Main exposing (bar, multilineList, ratio)
 
 
 ratio =
-    graphHeight
-        / (if range == 0 then
-            0.1
-
-           else
-            toFloat range
-          )
+    graphHeight / (if range == 0 then 0.1 else toFloat range)
```

Comment:
Why did the examples in this file need to change?  Was it just to add test coverage of the new single-line case? Can these go back to how they were after the new cases are added to `Elm-0.19/AllSyntax/Expressions.elm`?

Reply:
Oh I see, the input files for these tests were single-line.  That seems fine, then.